rrmngmnt/ssh.py

```diff
@@ -234,3 +237,55 @@ def run_cmd(self, cmd, input_=None, tcp_timeout=None, io_timeout=None):
         """
         with self.session(tcp_timeout) as session:
             return session.run_cmd(cmd, input_, io_timeout)
+
+    def is_connective(self, tcp_timeout=20.0):
+        """
+        Check if address is connective via ssh
+
+        :param tcp_timeout: time to wait for response
+        :type tcp_timeout: float
+        :return: True if address is connective, False otherwise
+        :rtype: bool
+        """
+        try:
+            self.logger.info(
+                "Check if address %s is connective via ssh", self.address
```

Comment:
self.address is already appended by LoggerAdapter, so it seems to be redundant to me